File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SlimeRobe.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.LightArmor;

/* JADX INFO: loaded from: classes3.dex */
public class SlimeRobe extends LightArmor {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.armor_light_slime_robe_name;
        this.idDescription = R.string.armor_light_slime_robe_description;
        this.idImage = R.drawable.slime_robe;
        this.price = 630L;
        this.maxHp = 80;
        this.defense = 5;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SlimeRobe.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.LightArmor

class SlimeRobe : LightArmor() {
    override fun configureProperties() {
        idName = R.string.armor_light_slime_robe_name
        idDescription = R.string.armor_light_slime_robe_description
        idImage = R.drawable.slime_robe
        price = 630L
        maxHp = 80
        defense = 5
    }
}
